### PR TITLE
Cleanup gitignore file

### PR DESCRIPTION
**Pull request by thomasbabuj**
I am targetting this branch, because this is a pedantic change.

## Subject

Removes platform entries and old unused files.

Refs: [sonata-project/dev-kit#211](https://github.com/sonata-project/dev-kit/issues/211)

### CODE REVIEW

**Comment by greg0ire** [comment]
I see no changes in the diff :confused:

File: .gitignore [hunk 1]
```diff
@@ -1,10 +1,9 @@
...
-vendor
+/build
+/Resources/doc/_build/
+/vendor
 .php_cs.cache
+composer.lock
+phpunit.xml
+
+
+
```
Comment (greg0ire) [inline .gitignore]: Please remove the extra lines

**Comment by thomasbabuj** [comment]
Hi @greg0ire, Extra lines removed. Sorry about that.

File: .gitignore [hunk 2]
```diff
@@ -1,10 +1,6 @@
...
-nbproject
-coverage
-composer.lock
-vendor
+/build
+/Resources/doc/_build/
+/vendor
 .php_cs.cache
+composer.lock
+phpunit.xml
```
Comment (greg0ire) [inline .gitignore]: Please configure your IDE to add a newline at end of file, it should make the red pictogram disappear

**Comment by thomasbabuj** [comment]
Included the newline at the end of file.

**Comment by greg0ire** [comment]
I asked @rande and @Soullivaneuh to fix the styleci build, it is stuck, sorry for that.

**Comment by core23** [comment]
Please rebase, this will retrigger StyleCI. After that we will merge this.

**Pull request merged**